api_end_end_testing/src/main/java/tds/user/api/model/RoleAssociation.java

```diff
@@ -0,0 +1,42 @@
+package tds.user.api.model;
```

Comment:
Sorry, that is what I meant.  Inside "test" instead of inside "main."  Bad wording on my part.

And no, I think Maven will be the one always running it.